docs/pages/database-access/auto-user-provisioning/aws-redshift.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: AWS Redshift Automatic User Provisioning 
+description: Configure automatic user provisioning for AWS Redshift.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="14.2"
+>
+  Automatic user provisioning for AWS Redshift is available starting from
+  Teleport 14.2.
+</Details>
+
+(!docs/pages/includes/database-access/auto-user-provisioning/intro.mdx!)
+
+## Prerequisites
+
+- Teleport cluster with a configured [AWS Redshift](../guides/postgres-redshift.mdx) database.
+- Ability to connect to and create user accounts in the target database.
+
+<Admonition type="note" title="Supported services">
+Automatic user provisioning is not compatible with Redshift Serverless.
+</Admonition>
+
+## Step 1/3. Configure database admin
+
+Teleport uses the same authentication mechanism (IAM authentication) when
+connecting as an admin user as for regular user connections.
+
+The admin user must have privileges within the database to create users and
+grant them privileges. The admin user must also have privileges to monitor user
+processes and role assignments:
+```
+CREATE USER "teleport-admin" WITH PASSWORD DISABLE;
+GRANT ROLE "sys:superuser" TO "teleport-admin";
+```
+
+Users created by Teleport will be assigned the `teleport-auto-user` role in the
+database, which will be created automatically if it doesn't exist.
+
+Teleport will not delete the automatically created user at the end of the
```

Comment:
This might be already covered by the "Create Teleport role" section ([which was updated here](https://github.com/gravitational/teleport/pull/33819/files#diff-dc26e09b03e5c76d48d6867ccf16c543df3b163b00a84216bb9e83058297bc40R29)).

Reply:
thank you 🙏 !